### PR TITLE
[UPD] .travis.yml from maintainer-quality-tools

### PR DESCRIPTION
**Pull request by hbrunn**
I already added the dependency to web because newsletter needs it

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,46 +1,26 @@
...
+  - "2.7"
 
 env:
-    - ODOO="https://github.com/savoirfairelinux/odoo/archive/setuptools-addons.tar.gz"  # Temp until https://github.com/odoo/odoo/issues/185 or https://github.com/odoo/odoo/issues/441 is fixed
-#    - ODOO="https://github.com/odoo/odoo/archive/master.tar.gz"
-#    - ODOO="https://github.com/OCA/OCB/archive/master.zip"
+  - LINT_CHECK="1"
+  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
+  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+  - VERSION="8.0" ODOO_REPO="odoo/odoo" UNIT_TEST="1" LINT_CHECK="0"
```
Comment (pedrobaeza) [inline .travis.yml]: This is very slow when the number of modules are medium-high

Comment (hbrunn) [inline .travis.yml]: To be honest, I'm not sure why we need that when we have the two lines before. That's how it's proposed in https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml, but I'm happy to remove this if you agree

Comment (pedrobaeza) [inline .travis.yml]: It was proposed to make isolated test for each module to check dependencies between them. You can remove it with no problem.

**Comment by yvaucher** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by dreispt** [comment]
:+1:

**Comment by rvalyi** [comment]
:+1:

**Pull request merged**